src/translations/en.json

```diff
@@ -1103,6 +1103,20 @@
           "no_aliases": "Configure aliases and expose settings for voice assistants"
         }
       },
+      "device-registry-detail": {
+        "name": "[%key:ui::panel::config::devices::name%]",
+        "enabled_label": "[%key:ui::panel::config::devices::enabled_label%]",
+        "enabled_cause": "[%key:ui::panel::config::devices::enabled_cause%]",
+        "type": {
+          "device_heading": "[%key:ui::panel::config::devices::type::device_heading%]",
+          "device": "[%key:ui::panel::config::devices::type::device%]",
+          "service_heading": "[%key:ui::panel::config::devices::type::service_heading%]",
+          "service": "[%key:ui::panel::config::devices::type::service%]"
+        },
+        "enabled_description": "[%key:ui::panel::config::devices::enabled_description%]",
+        "update": "[%key:ui::panel::config::devices::update%]",
+        "unknown_error": "[%key:ui::panel::config::devices::unknown_error%]"
```

Comment:
Are these keys used on the device page? Otherwise we should just delete them there

Reply:
I didn't check, but I also thought that if we just give these new keys it will require them all to be retranslated. If we use references, than I think they don't need to be retranslated? Should I give any weight to that?